beacon-chain/sync/rpc_status.go

```diff
@@ -222,6 +222,11 @@ func (s *Service) statusRPCHandler(ctx context.Context, msg interface{}, stream
 			}
 			// Close before disconnecting, and wait for the other end to ack our response.
 			closeStreamAndWait(stream, log)
+			forkDigest, err := s.currentForkDigest()
```

Comment:
useful for debugging if we get into this error condition.

Reply:
This is going to be very noisy, especially if we connect with peers from different testnets. Have you checked to see if mainnet nodes start becoming noisy with this particular added log ?

Reply:
good point re mainnet, I'll remove it.